danilovid/second/task1.py.py

```diff
@@ -0,0 +1,12 @@
+"Використовуючи регулярні вирази, для поданого нижче тексту замініть кожне входження автомобільного номеру у форматі `АА 0000 BB` на текст `[номер АА]`. Виведіть результат."
+
+import re
+
+print("Завдання 5")
+
+f1 = input("Введіть перші два елементи: ")
+f2 = input("vvedite chisla: ")
+f3 = input("Введіть перші два елементи: ")
+stroka = (f1+f2+f3)
+print(stroka)
+print(re.compile((re.findall((r'\w{2}\s\d{4}\w{2}',"AA 0000 BB"))),'[nomer AA]'))
```

Comment:
Also to add regular expressions into groups in order to get data. And then print them.